spec/models/refinery/jobs/job_spec.rb

```diff
@@ -16,6 +22,40 @@ module Jobs
           expect(FactoryGirl.build(:job, {description: 1, title: 1})).to be_valid
         end
       end
+
+      context 'update' do
+        it 'should be functional' do
+          gen_title = generate_string
+          gen_description = generate_string
+
+          @job.update_attributes({title: gen_title, description: gen_description })
+
+          expect(@job.title).to eq gen_title
+          expect(@job.description).to eq gen_description
+        end
+      end
+
+      context 'destroy' do
+        it 'should be functional' do
+          job_id = @job.id
+          job_app_id = @job_application.id
+          @job_application.update_attributes({job: @job})
+          @job.destroy
+
+          expect(Refinery::Jobs::JobApplication.find_by_id(job_app_id)).to be_nil
+          expect(Refinery::Jobs::Job.find_by_id(job_id)).to be_nil
+        end
+      end
+
+      context 'latest' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.